std/stdio.d

```diff
@@ -1897,6 +1897,28 @@ $(CONSOLE
         assert(b1 == true && b2 == false);
     }
 
+    @system unittest
+    {
+        static import std.file;
+
+        auto deleteme = testFilename();
+        std.file.write(deleteme, "1\n2");
+        scope(exit) std.file.remove(deleteme);
+        int input;
+        auto f = File(deleteme);
+        f.readf("%s", &input);
+
+        import std.conv : ConvException;
+        try
+        {
+            f.readf("%s", &input);
```

Comment:
@wilzbach looks like six versus half a dozen - with collectException you need to pass an int by ref etc. I'll allow this.